examples/06-dc/read_plot_DC_data_wit_IO_class.py

```diff
@@ -0,0 +1,70 @@
+"""
+Reading and Plotting data with DC.IO class
+==========================================
+
+The DC.IO class is a convenient way to handle DC data and
+carry inversions within a same class. It also has several plotting utils
+such as pseudosections. We show here an example of plotting DC data based
+on a demonstration dataset.
+"""
+
+import numpy as np
+import pandas as pd
+import shutil
+import os
+from SimPEG import DC
+
+
+def run(PlotIt=True):
```

Comment:
you don't need to do this inside of the def run. It is okay, and we can merge as-is, but the notebook-styled examples are nicer: e.g. http://docs.simpeg.xyz/content/examples/07-fdem/plot_loop_loop_2Dinversion.html#sphx-glr-content-examples-07-fdem-plot-loop-loop-2dinversion-py

https://github.com/simpeg/simpeg/blob/master/examples/07-fdem/plot_loop_loop_2Dinversion.py. 

If you are game to update this before we get the pr in, that would be appreciated :)

Reply:
Will do